### PR TITLE
[UIPO-58] Create Blog Filter Component

### PR DESCRIPTION
**Pull request by KhrystynaKovtun**

### CODE REVIEW

File: src/components/BlogPage/BlogPage.js [hunk 1]
```diff
@@ -15,29 +19,47 @@ class BlogPage extends React.Component {
...
+
     const userAPI = new HttpService();
-    const blogsURL = `${process.env.SERVER_URL}/blogs`;
+    const blogsURL = `${process.env.SERVER_URL}/blogs${query}`;
     const { getBlogItemsSuccess, getBlogItemsError } = this.props;
 
     try {
       const response = await userAPI.get(blogsURL);
 
+      setTimeout(() => this.setState({ isLoading: false }), 500);
```
Comment (Hollister009) [inline src/components/BlogPage/BlogPage.js]: Why are you using setTimeout for state update?

Comment (KhrystynaKovtun) [reply src/components/BlogPage/BlogPage.js]: Because I want to take some time showing preloader

Comment (KhrystynaKovtun) [reply src/components/BlogPage/BlogPage.js]: Because I want to take some time showing preloader

Comment (Hollister009) [reply src/components/BlogPage/BlogPage.js]: Well, can you move to function then, and give it a meaningful name?

File: src/components/BlogPage/BlogPage.spec.js [hunk 2]
```diff
@@ -1,32 +1,55 @@
...
+import axios from "axios";
+import { shallow } from "enzyme";
 
-import BlogPage from './BlogPage';
-import mockedData from '../../mockedDataForTests';
+import BlogPage from "./BlogPage";
+import mockedData from "../../mockedDataForTests";
 
-describe('<BlogPage />', () => {
+describe("<BlogPage />", () => {
```
Comment (Hollister009) [inline src/components/BlogPage/BlogPage.spec.js]: What is the rule for quotes in your **IDE** ?

**Comment by Hollister009** [comment]
@KhrystynaKovtun please consider my comment

File: src/components/BlogPage/BlogPage.spec.js (hunk 2 above)
Comment (KhrystynaKovtun) [reply src/components/BlogPage/BlogPage.spec.js]: it is doing the prettier extension, it should be a single quote?

Comment (Hollister009) [reply src/components/BlogPage/BlogPage.spec.js]: Looks like that:
`"prettier.singleQuote": true`

**Review by Hollister009 — APPROVED** [review]
Approved. Nice work!

**Pull request merged**